src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/DelegatingVideoLinkBookingEventListener.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services.court
```

Comment:
There are two `VideoLinkBookingEventListener`s.  One for application insights and one to store the event in video_link_booking_event table.  This class just forwards events to all the listeners injected by Spring (those qualified by 'delegate').  This listener, being the @Primary listener, is the one that is injected into CourtService.